### PR TITLE
More Feats

### PR DESCRIPTION
**Pull request by ThyWoof**

### CODE REVIEW

**Comment by ThyWoof** [comment]
This request touches translations which is holding other activities I was willing to do. Appreciated if we get into a consensus here.

File: SolastaCommunityExpansion/Feats/FeatsValidations.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+namespace SolastaCommunityExpansion.Feats
+{
+    internal static class FeatsValidations
+    {
+        //
+        // validation routines for FeatDefinitionWithPrerequisites
+        //
+
+        internal static Func<FeatDefinition, RulesetCharacterHero, (bool result, string output)> ValidateMinCharLevel(int minCharLevel)
```
Comment (PhilPJL) [inline SolastaCommunityExpansion/Feats/FeatsValidations.cs:14]: minCharLevel isn't used.

File: SolastaCommunityExpansion/Patches/CustomFeatures/FeatDefinitionWithPrereqs/GuiFeatDefinitionPatcher.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            ref string prerequisiteOutput)
+        {
+            if (feat is not FeatDefinitionWithPrerequisites featDefinitionWithPrerequisites || featDefinitionWithPrerequisites.Validators.Count == 0)
+            {
+                return;
+            }
+
+            var (result, output) = featDefinitionWithPrerequisites.Validate(featDefinitionWithPrerequisites, hero);
+
+            __result &= result;
```
Comment (PhilPJL) [inline SolastaCommunityExpansion/Patches/CustomFeatures/FeatDefinitionWithPrereqs/GuiFeatDefinitionPatcher.cs:24]: I think that should be ___result &&= result;_

File: SolastaCommunityExpansion/Feats/FeatsValidations.cs (hunk 1 above)
Comment (ThyWoof) [reply SolastaCommunityExpansion/Feats/FeatsValidations.cs:14]: it is on 22 and 26 but indeed there was still a hard coded 4 there on 18.

Comment (PhilPJL) [reply SolastaCommunityExpansion/Feats/FeatsValidations.cs:14]: that's what I meant

Comment (ThyWoof) [reply SolastaCommunityExpansion/Feats/FeatsValidations.cs:14]: I read the comment on &= but you removed. Confirmed &= was working but changed to A = A && B.

**Review by PhilPJL — APPROVED** [review]
I'm ok with this approach.
I haven't tested the feats in game - no time.

**Pull request merged**